pop-api/src/v0/fungibles/events.rs

```diff
@@ -13,6 +13,7 @@ use super::*;
 /// Event emitted when allowance by `owner` to `spender` changes.
 // Differing style: event name abides by the PSP22 standard.
 #[ink::event]
+#[derive(Debug)]
```

Comment:
These features are only necessary for tests and therefore we don't want to include this in production contracts:
`#[cfg_attr(feature = "std", derive(Debug))]`

Reply:
Thanks!